enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/models/SubscriptionPolicy.java

```diff
@@ -78,17 +80,18 @@ public void setGraphQLMaxDepth(int graphQLMaxDepth) {
     }
     @Override
     public String getCacheKey() {
-
-        return PolicyType.SUBSCRIPTION + SubscriptionDataStoreUtil.getPolicyCacheKey(getName());
-
+        String org = StringUtils.isNotEmpty(getOrganization()) ? getOrganization() :
+                APIConstants.SUPER_TENANT_DOMAIN_NAME;
```

Comment:
What if we update the getOrganization() method this logic?
So here we can just getOrganization() and inside getOrganization we will return SUPER_TENANT_DOMAIN_NAME if org is empty.

```suggestion
        String org = getOrganization()
```

Reply:
Is it possible to org be empty?

Reply:
We discussed this earlier, too. Integration tests don't have an org id.